scripts/bin/cps_get_oid.py

```diff
@@ -34,7 +34,7 @@
         o = cps_object.CPSObject(obj=entry)
         if (_cur_key!=o.get_key()):
             _cur_key = o.get_key()
-            print('\n============%s==========\n' % cps.name_from_key(_cur_key,1))
+            print('\n============%s==========' % _cur_key)
```

Comment:
Please revert this change. It was fixed in https://github.com/open-switch/opx-cps/commit/27c5aa1574abfc592a519383f054659f6bb7bfd1